main.js

```diff
@@ -123,14 +150,15 @@ export async function embed(el, options) {
 
   // Little bit of a hacky way to find the inner <table> element.
   observeTableChangesAndFormatColumnCell(dt, {
-    columnIndex: options.mode === "obs" ? 1 : 11,
-    format(v) {
-      let a = document.createElement("a");
-      a.innerText = v;
-      a.href = `https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=${v}`;
-      a.target = "_blank";
-      a.style.fontVariantNumeric = "tabular-nums"; // looks better with monospacing
-      return a;
+    format: {
+      observation_id: (v) => {
+        let a = document.createElement("a");
+        a.innerText = v;
+        a.href = `https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=${v}`;
+        a.target = "_blank";
+        a.style.fontVariantNumeric = "tabular-nums"; // looks better with monospacing
+        return a;
```

Comment:
@sbooeshaghi 

You should be able to add additional functions (keyed by their column name) here to reformat other columns